.github/workflows/pytest.yml

```diff
@@ -185,7 +185,7 @@ jobs:
       - name: Run integration tests, trying to use GCS cache if possible
         run: |
           pip install --no-deps --editable .
-          pudl_datastore --dataset epacems --partition year_quarter=2022q1
+          pudl_datastore --dataset epacems --partition year_quarter=2023q1
```

Comment:
We forgot to update this when switching to the most recent year of epacems data in the fast ETL recently.  And actually... shouldn't this be like, 2025q1 now?

Reply:
yes, 2025q1 is the most recent!

Reply:
I saw some chatter elsewhere about standardizing the instructions for what data gets run through the fast ETL, and I think it should probably always include the most recently available partition -- so that the thing that's actually being mucked with when we're doing updates gets exercised in the CI.  But we've drifted away from this across the board and we should probably have a PR that just goes in and updates all of the fast ETL settings to use the most recent partitions.  But we can do that after the release and after we get fast row counts working again.